helper/impls/helper-impl-16/src/main/java/com/linkedin/avroutil1/compatibility/avro16/Avro16Adapter.java

```diff
@@ -203,6 +203,11 @@ public Object getGenericDefaultValue(Schema.Field field) {
     return Avro16DefaultValuesCache.getDefaultValue(field, false);
   }
 
+  @Override
+  public boolean fieldHasDefault(Schema.Field field) {
+    return null != field.defaultValue();
```

Comment:
what if the default value is null?

Reply:
Then the `field.defaultValue` should be a `NullNode`, and thus not `null`.